<?xml version="1.0"?>
<launch>
    <!-- Params -->
    <rosparam file="$(find ee4308_hector)/config/hector.yaml" />
    <param name="initial_x" value="$(env HECTOR_X)" type="double"/>
    <param name="initial_y" value="$(env HECTOR_Y)" type="double"/>
    <param name="initial_z" value="$(env HECTOR_Z)" type="double"/>
    <rosparam command="load" file="$(find drone_commander)/config/drone_commander.yaml"/>
    <rosparam command="load" file="$(find drone_commander)/config/trajectory_generator.yaml"/>
    <rosparam command="load" file="$(find drone_commander)/config/velocity_controller.yaml"/>
    <param name="run" value="true" type="bool"/>

    <!-- Nodes -->
    <node pkg="drone_commander" type="drone_commander_node" name="drone_commander" output="screen" />
    <node pkg="ee4308_hector" type="hector_motion" name="hector_motion" output="screen" />
</launch>
